src/components/Consent.js

```diff
@@ -17,10 +17,10 @@ export const Consent = () => {
   useSoftkey('ConsentMessage', {
     center: isOnline ? i18n('softkey-consent-agree') : '',
     onKeyCenter: onAgree,
-    left: i18n('softkey-consent-terms'),
-    onKeyLeft: goto.termsOfUse,
-    right: i18n('softkey-consent-policy'),
-    onKeyRight: goto.privacyPolicy,
+    left: isOnline ? i18n('softkey-consent-terms') : '',
```

Comment:
Since almost all the keys are conditional, you could consider having 2 different objects: one fully configured and one with a bunch of blank keys and decide which one to pass to `useSoftkey` based on `isOnline`.

Reply:
Thanks! You remind me how we handle this is onboarding page with the replace params.

Reply:
Nice